src/main/java/xyz/nucleoid/plasmid/game/ManagedGameSpace.java

```diff
@@ -338,6 +341,9 @@ public void close(GameCloseReason reason) {
 
         Scheduler.INSTANCE.submit(server -> {
             try {
+                // TODO: Hook game closing event here
+                GameEvents.CLOSING.invoker().onGameClosing(this, reason);
```

Comment:
Is this TODO still relevant? This can also be renamed to `onGameSpaceClosing` for consistency